### DISCUSSION TITLE
Quantity Formatter

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#2960] [Q&A] [answered]
I  have defined Quantity Formatter ,

![image](https://user-images.githubusercontent.com/53996840/147680498-772cbbb4-b774-4173-a885-f3628fbdba8b.png)

while using the Measure Distance Tool , I get value as 
![image](https://user-images.githubusercontent.com/53996840/147680639-b11cef31-3033-4790-b048-52cdeb710af0.png)

I expect the Value as  9,762.25 
why the ThousandSeparator( "," ) not coming here though I have defined ??

**Comment by bbastings** [comment, marked as answer]
This is not my area, but since folks are on vacation, try this:

formatTraits: ["keepSingleZero", "showUnitLabel", "use1000Separator"],
